getting_started/documentation.rst

```diff
@@ -223,18 +223,39 @@ duplicate your instance in 10 minutes, and more!
 Support
 ~~~~~~~
 
-Your Odoo Online subscription includes an **unlimited support service at
-no extra cost, 24/5, Monday to Friday**. To cover 24 hours, our teams
-are in San Francisco, Belgium, and India. Questions could be about
-anything and everything, like specific questions on current Odoo features and where to configure them, bugfix requests,
-payments, or subscription issues.
+Your Odoo Online subscription includes **unlimited 24hr support at no extra  
+cost, Monday to Friday**. Our teams are located around the world to ensure you have support,
+no matter your location. Your support representative could be communicating to you from 
+San Francisco, Belgium, or India!
+
+What kind of support is included?
+
+-    Providing you with relevant material (guidelines, product documentation,                                                                etc...)
+-    Answers to issues that you may encounter in your standard Odoo database                                                                  (eg. “I cannot close my Point of Sale” and  “How do I measure my sales KPIs?”)
```

Comment:
> How do I measure my sales KPIs?
how is that an issue? it does not fall under the "Training on how to use our software (we will direct you to our many resources) " ?